### PR TITLE
[13.0] [MIG] website_oca_psc_team: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT23216

### CODE REVIEW

**Comment by CarlosRoca13** [comment]
Please @pedrobaeza @ernestotejeda review this

File: website_oca_psc_team/templates/psc_category_template.xml [hunk 1]
```diff
@@ -59,8 +59,8 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
             >
                 <div class="o_web_psc_team_project_detail">
                     <img
-                        t-att-src="'data:image/png;base64,%s'%to_text(project.image_medium)
-                         if project.image_medium else 'website_oca_psc_team/static/src/img/icon.png'"
+                        t-att-src="'data:image/png;base64,%s'%to_text(project.image_256)
+                         if project.image_256 else 'website_oca_psc_team/static/src/img/icon.png'"
```
Comment (pedrobaeza) [inline website_oca_psc_team/templates/psc_category_template.xml]: `image_medium` was previously `128x128 image` and you unify the calls to the same field, caching it. At the end, all of them are resized at client (browser) level due to `height` and `width` attributes, and all occurrences are lower than 128x128.
```suggestion
                        t-att-src="'data:image/png;base64,%s'%to_text(project.image_128)
                         if project.image_128 else 'website_oca_psc_team/static/src/img/icon.png'"
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@pedrobaeza change done!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**